packages/html-reporter/src/testCaseView.tsx

```diff
@@ -32,13 +32,20 @@ export const TestCaseView: React.FC<{
 }> = ({ projectNames, test, run, anchor }) => {
   const [selectedResultIndex, setSelectedResultIndex] = React.useState(run);
 
+  const annotations = new Map<string, (string | undefined)[]>();
+  test?.annotations.forEach(annotation => {
+    const list = annotations.get(annotation.type) || [];
+    list.push(annotation.description);
```

Comment:
Might be worth to force string here, otherwise if an object is passed by mistake as description, the html report won't render with no indication of what the problem is.

```javascript
    list.push(new String(annotation.description));
```